nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -335,6 +335,7 @@ def run_single_instance(config):
     config["workers"] = 4
     config["log_level"] = "INFO"
     config["reuse_actors"] = False
+    config["dist_port"] = get_free_port()
```

Comment:
This is for the run_single_instance only, not for the distributed. The get free port function from Ray has issues in the single instance case, if you break an experiment midway it will try to reuse the same port and fail.

Reply:
Got it.

Reply:
You may need to change this line as well.
https://github.com/numenta/nupic.research/blob/f896cbd4e2b718c88059309c166cf9713d2dc237/nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py#L371